src/frontend/src/flows/manage/deviceSettings.ts

```diff
@@ -136,15 +136,24 @@ const deleteDevice = async ({
     pubKey
   );
 
-  const shouldProceed = sameDevice
-    ? confirm(
-        "This will remove your current device and you will be logged out."
-      )
-    : confirm(
-        `Do you really want to remove the ${
-          isRecoveryDevice(device) ? "" : "device "
-        }"${device.alias}"?`
-      );
+  // Different confirmation based on the device
+  const confirmationPrompt = [];
+  if (isRecoveryDevice(device)) {
+    confirmationPrompt.push("Remove your Recovery Device");
+    confirmationPrompt.push(
+      "Are you sure you want to remove your recovery device? You will no longer be able to use it to recover your account."
+    );
+  } else {
+    confirmationPrompt.push(
+      `Do you really want to remove the device "${device.alias}"?`
+    );
+  }
+  if (sameDevice) {
```

Comment:
it's correct that the message can be one of the above + this message - i.e. not `else if (sameDevice)`?

Reply:
Yep!